### PR TITLE
Step 1 /change-or-apply endpoint

### PR DESCRIPTION
**Pull request by karpet**
Please merge #90 first and then I will rebase this branch.

Depends on https://github.com/BlueprintKansas/ksmyvoteinfo-py/pull/2 to be merged and released before tests will build.

### CODE REVIEW

File: app/services/steps/step_0.py [hunk 1]
```diff
@@ -17,11 +18,11 @@ def run(self):
 		if not self.verify_form_requirements():
 			return False
 
-		reg_lookup = self.lookup_registration(name_first=self.form_payload['name_first'], name_last=self.form_payload['name_last'], dob=self.form_payload['dob'], county=self.form_payload['county'])
+		self.reg_found = self.lookup_registration(name_first=self.form_payload['name_first'], name_last=self.form_payload['name_last'], dob=self.form_payload['dob'], county=self.form_payload['county'])
```
Comment (karpet) [inline app/services/steps/step_0.py]: keep this around so we can display it

**Comment by JoshuaMitchell** [comment]
@karpet I can't merge these in there is a build error with the new ksvotemyinfo package?  Any ideas?

> No matching distribution found for ksmyvoteinfo==0.3 (from -r requirements.txt (line 22))
> make: *** [deps] Error 1
> The command "make deps" failed and exited with 2 during .

**Comment by karpet** [comment]
@JoshuaMitchell as I mentioned in the description, must merge and release https://github.com/BlueprintKansas/ksmyvoteinfo-py/pull/2 first

**Comment by JoshuaMitchell** [comment]
@karpet I did merge in 90, but I don't have write access to the ksmyvoteinfo repo to merge that in.  I think that will need to be pushed up to pypi too?

**Comment by karpet** [comment]
@JoshuaMitchell all 💚 now - ready to review

**Pull request merged**